sshclient.go

```diff
@@ -65,6 +65,29 @@ func DialWithKey(addr, user, keyfile string) (*Client, error) {
 	return Dial("tcp", addr, config)
 }
 
+// DialWithKeyWithPassphrase same as DialWithKey but with a passphrase to decrypt the private key
+func DialWithKeyWithPassphrase(addr, user, keyfile string, passphrase []byte) (*Client, error) {
```

Comment:
'passphrase' argument should as string type, just like your example, because we are passing it a string instead of raw bytes, usually.

Reply:
@helloyi thanks, done :)